extensions/hibernate-validator/spi/src/main/java/io/quarkus/hibernate/validator/spi/BeanValidationAnnotationsBuildItem.java

```diff
@@ -0,0 +1,35 @@
+package io.quarkus.hibernate.validator.spi;
+
+import java.util.Collections;
+import java.util.Set;
+
+import io.quarkus.builder.item.SimpleBuildItem;
+
+/**
+ * BuildItem used to publish the list of detected Bean Validation annotations
+ * for consumption by other extensions.
+ */
+public final class BeanValidationAnnotationsBuildItem extends SimpleBuildItem {
+
+    private final String valid;
+    private final Set<String> constraints;
+    private final Set<String> all;
+
+    public BeanValidationAnnotationsBuildItem(String valid, Set<String> constraints, Set<String> all) {
+        this.valid = valid;
+        this.constraints = Collections.unmodifiableSet(constraints);
+        this.all = Collections.unmodifiableSet(all);
+    }
+
+    public String getValid() {
```

Comment:
Maybe make the name a bit more explicit. I understand it because it matches my use case, but might be obscure for others.

What about `getValidAnnotation` and `getConstraintAnnotations`

Reply:
I refrained from doing that because of the build item name with contains Annotations. But I can add it if you think it makes things clearer!

Reply:
Yeah, thinking about it a bit more, I will do it.

Reply:
Done!